example/webdriverio-test/test/specs/test.e2e.ts

```diff
@@ -2,78 +2,202 @@ import {driver} from '@wdio/globals';
+    await typeToInput(driver, '~' + URL_INPUT, webSocketUrl);
+    await compareInputValue(driver, '~' + TOKEN_INPUT, peerDetail?.token);
+    await compareInputValue(driver, '~' + URL_INPUT, webSocketUrl);
   });
-  it('connect, request necesary permissions and move to room', async () => {
+  it('request necesary permissions and connect', async () => {
     await tapButton(driver, '~' + CONNECT_BUTTON);
-
     if (driver.isIOS) {
       await driver.acceptAlert();
       await tapApp(driver);
       await tapButton(driver, '~' + CONNECT_BUTTON);
       await driver.acceptAlert();
     }
   });
-  it('toggle camera', async () => {
+  it('toggle off preview camera and microphone then join the room', async () => {
+    await tapButton(driver, '~' + TOGGLE_MICROPHONE_BUTTON);
     await tapButton(driver, '~' + TOGGLE_CAMERA_BUTTON);
+    await tapButton(driver, '~' + JOIN_BUTTON);
+  });
+  it('check if no camera view', async () => {
+    await getElement(driver, '~' + NO_CAMERA_VIEW);
+  });
+  it('toggle camera on', async () => {
     await tapButton(driver, '~' + TOGGLE_CAMERA_BUTTON);
   });
+  it('check if only one video cell', async () => {
+    await getElement(driver, '~' + VIDEO_CELL + 0);
+    await getElement(driver, '~' + VIDEO_CELL + 1, false);
+  });
   it('switch camera', async () => {
     await tapButton(driver, '~' + SWITCH_CAMERA_BUTTON);
     await tapButton(driver, '~' + SWITCH_CAMERA_BUTTON);
   });
-
-  it('share screen', async () => {
+  it('screen share on', async () => {
     await tapButton(driver, '~' + SHARE_SCREEN_BUTTON);
-
     if (driver.isAndroid) {
       await tapButton(driver, '//*[@text="Start now"]');
-      await driver.pause(2000);
-      await tapButton(driver, '~' + SHARE_SCREEN_BUTTON);
     } else {
       await tapButton(
         driver,
         '//XCUIElementTypeButton[@name="Start Broadcast"]',
       );
       await tapApp(driver);
-      await driver.pause(3000);
-      await tapButton(driver, '~' + SHARE_SCREEN_BUTTON);
       await tapButton(
         driver,
         '//XCUIElementTypeButton[@name="Stop Broadcast"]',
       );
-      await tapApp(driver);
     }
-
+  });
+  it('check if two video cells', async () => {
+    await getElement(driver, '~' + VIDEO_CELL + 0);
+    await getElement(driver, '~' + VIDEO_CELL + 1);
+    await getElement(driver, '~' + VIDEO_CELL + 3, false);
+  });
+  it('toggle video statistics on ', async () => {
+    await tapButton(driver, '~' + TOGGLE_STATISTICS_BUTTON);
+    await tapButton(driver, '~' + VIDEO_STATISTICS_RTC);
+  });
+  it('toggle video statistics off ', async () => {
+    await tapButton(driver, '~' + VIDEO_STATISTICS_RTC);
+    await tapButton(driver, '~' + TOGGLE_STATISTICS_BUTTON);
+  });
+  it('toggle camera off', async () => {
+    await tapButton(driver, '~' + TOGGLE_CAMERA_BUTTON);
+    await getElement(driver, '~' + NO_CAMERA_VIEW);
+  });
+  it('check if only 1 video cell', async () => {
```

Comment:
nie testowałem tego, ale driver udostępnia pobieranie wielu elementów, można spróbować tak to osiągnąć:
const element = await driver.$$(selector+ "*");
jeżeli nie to to można podłubać w internetach

Reply:
szukalem nie moglem nic takiego znalesc dla rn 
jak znajdziesz cos dzialajacego to z checia tego uzyje 🥰